mantle/platform/qemu.go

```diff
@@ -139,7 +139,9 @@ type QemuBuilder struct {
 	// Config is a path to Ignition configuration
 	Config string
 
-	Memory     int
+	// Memory defaults to 1024 on most architectures, others it may be 2048
+	Memory int
+	// Processors < 0 means to use host count, unset means 1, values > 1 are directly used
 	Processors int
```

Comment:
what about:

Processors == 0 -> means auto detect
Processsors unset == '1'
Processors == N -> use N

Reply:
> Processors == 0 -> means auto detect

I guess that means we'd be changing current functionality?

Reply:
Yeah, maybe something to do later, don't want to roll in redoing the API as part of this change.